SoraFoundation/Classes/Timer/CountdownTimer.swift

```diff
@@ -64,11 +75,12 @@ public final class CountdownTimer: NSObject {
     }
 
     private func scheduleTimer() {
-        timer = Timer.scheduledTimer(timeInterval: 1.0,
+        timer = Timer.scheduledTimer(timeInterval: notificationInterval,
                                      target: self,
                                      selector: #selector(actionTimer(_:)),
                                      userInfo: nil,
                                      repeats: true)
+        RunLoop.main.add(timer!, forMode: .common)
```

Comment:
Please, add run loop  and modes parameters to `start` function. Also put current old start function to extension and call new one with Runloop.main and .default mode. This saves backward compatibility.